packages/cms/store/auth.ts

```diff
@@ -46,7 +55,10 @@ export const useAuthStore = create<AuthState & AuthMethods>(
       return window.Tensei.api.post('/auth/register', credentials)
     },
     async logout() {
-      return window.Tensei.api.post('/auth/logout')
+      return window.Tensei.api.post('/logout')
```

Comment:
Actually the backend endpoint for this has changed. Made that change today to `/auth/logout`

Reply:
okay, would update it